superset/migrations/versions/a33a03f16c4a_add_extra_column_to_savedquery.py

```diff
@@ -1,3 +1,19 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
ditto

Reply:
would rather not split the PR, not sure why CI allowed these files to slip in without licenses, I'm guessing they come from branching before the license check

Reply:
It shouldn't be too hard to just unstage this file, right? :) 
Say if we cut the next apache release after this commit, it wouldn't be right to have this migration there as it is unrelated to the change (and unrelated to the release).